app.js

```diff
@@ -194,6 +180,9 @@ function updateShows () {
 
```

Comment:
In order to have all the previous data in your `junkClicks` and `junkViews` arrays, these functions are where you should be retrieving the data from `localStorage` and using `JSON.parse()` to make it usable.